api/src/pcapi/core/offerers/api.py

```diff
@@ -887,3 +888,141 @@ def get_offerer_offers_stats(offerer_id: int) -> sa.engine.Row:
     )
 
     return db.session.execute(offers_stats_query).one()
+
+
+def get_venue_basic_info(venue_id: int) -> sa.engine.Row:
+    dms_application_id_query = sa.select(finance_models.BankInformation.applicationId).filter(
+        finance_models.BankInformation.venueId == venue_id
+    )
+    has_bank_informations_query = sa.select(sa.func.count(offerers_models.VenueReimbursementPointLink.id)).filter(
+        offerers_models.VenueReimbursementPointLink.venueId == venue_id,
+        offerers_models.VenueReimbursementPointLink.timespan.contains(datetime.utcnow()),
+    )
+    venue_query = (
+        sa.select(
+            offerers_models.Venue.id,
+            offerers_models.Venue.publicName,
+            offerers_models.Venue.name,
+            offerers_models.Venue.siret,
+            offerers_models.VenueContact.email,
+            offerers_models.VenueContact.phone_number,
+            offerers_models.Venue.postalCode,
+            offerers_models.Venue.dmsToken,
+            offerers_models.Venue.venueEducationalStatusId,
+            dms_application_id_query.scalar_subquery().label("dms_application_id"),
+            has_bank_informations_query.scalar_subquery().label("has_bank_informations"),
+        )
+        .outerjoin(
+            offerers_models.VenueContact,
+        )
+        .filter(
+            offerers_models.Venue.id == venue_id,
+        )
+    )
+
+    venue = db.session.execute(venue_query).one_or_none()
+
+    return venue
+
+
+def get_venue_total_revenue(venue_id: int) -> float:
+    individual_revenue_query = sa.select(
+        sa.func.coalesce(
+            sa.func.sum(bookings_models.Booking.amount * bookings_models.Booking.quantity),
+            0.0,
+        )
+    ).filter(
+        bookings_models.Booking.venueId == venue_id,
+        bookings_models.Booking.status != bookings_models.BookingStatus.CANCELLED.value,
+    )
+    collective_revenue_query = (
+        sa.select(
+            sa.func.coalesce(
+                sa.func.sum(educational_models.CollectiveStock.price),
+                0.0,
+            )
+        )
+        .select_from(
+            educational_models.CollectiveBooking,
+        )
+        .join(
+            educational_models.CollectiveStock,
+            onclause=educational_models.CollectiveStock.id == educational_models.CollectiveBooking.collectiveStockId,
+        )
```

Comment:
(question de goût) Je trouve plus concis et plus aisé de lire la forme courte où l'on indique seulement la relation : 
```suggestion
        .join(educational_models.CollectiveBooking.collectiveStock)
```

Reply:
ça me va